src/incubator/TextField/CharCounter.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {useContext} from 'react';
+import {TextStyle, StyleSheet} from 'react-native';
+import _ from 'lodash';
+import Text from '../../components/text';
+import FieldContext from './FieldContext';
+
+export interface CharCounterProps {
+  showCharCounter?: boolean;
+  maxLength?: number;
```

Comment:
maxLength can be required (without the question mark), no?

Reply:
No. If we'll make it require people will always have to pass the `maxLength` prop.